examples/ssh/linux_updates_count.json

```diff
@@ -0,0 +1,43 @@
+{
+    "logo": "linux.svg",
+    "version": "1.0.1",
+    "name": "Linux Updates Count",
```

Comment:
Since the script is using apt, it will only work for debian based distributions. The wording here and below should most likely be adjusted.